docs/source/docker/advanced/datasource_predictions.rst

```diff
@@ -0,0 +1,347 @@
+.. _ref-docker-datasource-predictions:
+
+Add Predictions to a datasource
+===============================
+
+Lightly can not only use images you provided in a datasource, but also predictions of a ML model on your images.
+They are used for active learning for selecting images based on the objects in them.
+Furthermore, object detection predictions can be used running Lightly on object level.
+By providing the predictions in the datasource,
+you have full control over them and they scale well to millions of samples.
+Furthermore, if you add new samples to your datasource, you can simultaneously
+add their predictions to the datasource.
+If you already have labels instead of predictions, you can treat them
+just as predictions and upload them the same way.
+
+
+Predictions Folder Structure
+----------------------------
+
+In the following, we will outline the format of the predictions required by the
+Lightly docker. Everything regarding predictions will take place in a subdirectory
+of your configured datasource called `.lightly/predictions`. The general structure
+of this directory will look like this:
+
+
+.. code-block:: bash
+
+    datasource/my_dataset
+        + .lightly/predictions/
+            + tasks.json
+            + task_1/
+                 + schema.json
+                 + image_1.json
+                 ...
+                 + image_N.json
+            + task_2/
+                 + schema.json
+                 + image_1.json
+                 ...
+                 + image_N.json
+        + image_1.png
```

Comment:
Maybe it's just me but this seems hard to read. These images are in the `datasource/my_dataset` folder. Maybe put these files above the `.lightly` folder?